### PR TITLE
SET GITHUB_DEF_BR on demos 

### PR DESCRIPTION
**Pull request by vot4anto**

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+name: actions for env
+on:
+  push:
+    branches: actions
+  pull_request:
+    branches: master
```
Comment (micheles) [inline .github/workflows/actions.yml:7]: I think branches wants a list:
```
 branches: [actions]
 branches: [master]
```

Comment (vot4anto) [reply .github/workflows/actions.yml:7]: I read this guide: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

and one of example is the follow:
on:
  push:
    # Sequence of patterns matched against refs/heads
    branches:    
      # Push events on main branch
      - main
      # Push events to branches matching refs/heads/mona/octocat
      - 'mona/octocat'
      # Push events to branches matching refs/heads/releases/10
      - 'releases/**'

with one line for different branch or ref. 
But it seem that also the [] are allowed, for example:
 
Example using a list of events
# Triggers the workflow on push or pull request events
on: [push, pull_request]

Comment (micheles) [reply .github/workflows/actions.yml:7]: The example you quote says that
```
branches:
  -main
```
is correct, NOT `branches: main`

**Comment by micheles** [comment]
LGTM

**Pull request merged**